types/nova-editor-node/index.d.ts

```diff
@@ -19,25 +19,27 @@ type WritableStream<T = any> = unknown;
 type AssistantsRegistrySelector = string | { syntax: string };
 
 interface AssistantsRegistry {
-    registerColorAssistant(selector: AssistantsRegistrySelector, object: ColorAssistant): Disposable;
     registerCompletionAssistant(selector: AssistantsRegistrySelector, object: CompletionAssistant): Disposable;
     registerIssueAssistant(
         selector: AssistantsRegistrySelector,
         object: IssueAssistant,
-        options?: { event: "onChange" | "onSave" }
+        options?: { event: 'onChange' | 'onSave' },
     ): Disposable;
+    registerTaskAssistant(object: TaskAssistant, options?: { identifer: string; name: string }): Disposable;
```

Comment:
This should have been `identifier`, not `identifer`?

Reply:
yeah, looks like a typo